silverstripe/framework/SS-2014-017-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SS-2014-017: XML Quadratic Blowup Attack
+link:      http://www.silverstripe.org/software/download/security-releases/ss-2014-017-xml-quadratic-blowup-attack/
+cve:       ~
+branches:
+    3.1.x:
+        time:     2014-08-12 11:50:00
+        versions: [>=3.1.0,<=3.1.11]
```

Comment:
As far as I know it isn't, but I'll double check and open a new pull request if needed.